actors/miner/src/types.rs

```diff
@@ -243,6 +243,13 @@ pub struct CompactPartitionsParams {
     pub partitions: BitField,
 }
 
+#[derive(Serialize_tuple, Deserialize_tuple)]
+pub struct MovePartitionsParams {
```

Comment:
as mentioned above, I think the terminology in the FIP is clearer - can you adjust the PR to reflect?

Reply:
Done.